internal/database/strikes.go

```diff
@@ -0,0 +1,98 @@
+package database
+
+import (
+	"context"
+
+	"github.com/jackc/pgx/v5/pgxpool"
+)
+
+type AddStrikeEntryArgs struct {
+	DbUserID   int
+	Reason     string
+	StrikeTime string
+}
+
+func AddStrike(conn *pgxpool.Pool, p AddStrikeEntryArgs) (int, int, error) {
+	query := `INSERT INTO strikes (userID, reason, createTimestamp)
+	VALUES ($1, $2, $3)
+	RETURNING strikeID, (SELECT count(strikeID) from strikes WHERE userID = $1)`
+
+	var strikeID int
+	var strikeCount int
+	err := conn.QueryRow(
+		context.Background(),
+		query,
+		p.DbUserID,
+		p.Reason,
+		p.StrikeTime,
+	).Scan(&strikeID, &strikeCount)
+
+	if err != nil {
+		return -1, -1, err
+	}
+
+	// currently added strike is not counted when current strike count is returned
+	// Adding one here makes sure correct value is returned
+	return strikeID, strikeCount + 1, nil
+}
+
+type Strike struct {
+	StrikeID       int
+	Reason         string
+	DiscordUserID  string
+	DiscordGuildID string
```

Comment:
for now we can just use the constant `GuildID` as we're not planning on releasing this bot to other discord servers

Reply:
This just pulls from the user table, so I don't think it should ever be a different value, and since we already need to query users it doesn't add any real extra amount of overhead to just pull that info directly

Reply:
eh i guess it's fine, but i still don't really think we need a field for that since we're probably never going to extend this bot outside of NAUR

Reply:
fair, this is just printing anyways so it's easy to remove, done